src/components/UriViewer.js

```diff
@@ -52,7 +68,38 @@ const getMaxDownloadCostNA = bytes => {
   return downloadPrice < 0.01 ? '< $0.01' : Utils.formatUSD(downloadPrice)
 }
 
-const UriViewer = ajaxCaller(class UriViewer extends Component {
+const UriViewer = props => {
+  const signal = useCancellation()
+  const [valid, setValid] = useState(undefined)
```

Comment:
I remember thinking that we should rename this. Maybe something like `needsBillingProject` or `isRequesterPays`.